sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLListenerSuite.scala

```diff
@@ -36,13 +36,14 @@ import org.apache.spark.sql.catalyst.util.quietly
 import org.apache.spark.sql.execution.{LeafExecNode, QueryExecution, SparkPlanInfo, SQLExecution}
 import org.apache.spark.sql.execution.metric.{SQLMetric, SQLMetrics}
 import org.apache.spark.sql.test.SharedSQLContext
-import org.apache.spark.ui.SparkUI
+import org.apache.spark.status.config._
 import org.apache.spark.util.{AccumulatorMetadata, JsonProtocol, LongAccumulator}
-
+import org.apache.spark.util.kvstore.InMemoryStore
 
 class SQLListenerSuite extends SparkFunSuite with SharedSQLContext with JsonTestUtils {
   import testImplicits._
-  import org.apache.spark.AccumulatorSuite.makeInfo
+
+  override protected def sparkConf = super.sparkConf.set(LIVE_ENTITY_UPDATE_PERIOD, 0L)
```

Comment:
the spark context is shared for all test suites, we should only set this conf to 0 in this suite.

Reply:
Is that true? The suite extends `SharedSQLContext` (which extends `SharedSparkSession`) and `SQLTestUtils`, all of which are traits, not objects. (Unlike `TestHive` which does force sessions to be used across suites for hive tests.)

There are also other suites that modify the conf (such as `HDFSMetadataLogSuite`).

Reply:
ah you are right, it's only shared in hive tests